finder-app/manual-linux.sh

```diff
@@ -47,34 +53,71 @@ then
     sudo rm  -rf ${OUTDIR}/rootfs
 fi
 
-# TODO: Create necessary base directories
+echo "Creating root filesystem base directories"
+mkdir ${OUTDIR}/rootfs
+cd ${OUTDIR}/rootfs
+mkdir bin dev etc home lib proc sbin sys tmp usr var
+mkdir usr/bin usr/lob usr/sbin
+mkdir -p var/log
+
+#Setting ownership of entire rootfs to root
+sudo chown -R root:root *
```

Comment:
@ECleverito if you moved this step later you could probably run most of the steps below without `sudo`